### DISCUSSION TITLE
What is the color scheme showed in the screenshot in README?

### DISCUSSION THREAD

**Discussion by Explorare** [junegunn/fzf#3581] [General]
I did some search and read the wiki page, but still can't find the exact same color scheme.
![](https://raw.githubusercontent.com/junegunn/i/master/fzf-preview.png)

**Comment by junegunn** [comment]
It's seoul256-light.

* https://github.com/junegunn/seoul256.vim?tab=readme-ov-file#seoul256-light-version

You can find the `--color` options for that here: https://github.com/junegunn/fzf/blob/master/ADVANCED.md#color-themes

**Reply by Explorare** [reply]
Thank you very much.
